nationalarchives/ds-wagtail
# DF-681:Fix record opening year less than four digits

Ticket URL: https://national-archives.atlassian.net/browse/DF-681

## About these changes

Fix record opening year less than four digits

## How to check these changes

- Navigate to Catalogue search - TNA
- Enter Record opening from date - year = 19
- Enter Record opening from date - year = 200
- Click Update

The page should not show any errors.


## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
